include/error_handler.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <unordered_map>
+
+namespace vpd
+{
+/**
+ * @brief Class to support logging Error.
+ *
+ * This class provides framework to call 'create' method of 'phosphor-logging'
+ * service to log a PEL.
+ */
+class ErrorHandler
+{
+  public:
+    using ErrorData = std::unordered_map<std::string, std::string>;
```

Comment:
Why types.hpp is not being used in this case?

Reply:
moved to types.hpp